### DISCUSSION TITLE
No saving banks

### DISCUSSION THREAD

**Discussion by tizfra** [GeekFunkLabs/fluidpatcher#72] [Q&A]
Hi everyone,
after few months I'm been able to install fluidpatcher on my notebook (running windows 10).
Fluidpatcher runs perfectly and sounds comes out, but I can't add sounds in a bank and also I can't save modified bank.
What can I do?

**Comment by albedozero** [comment]
Help me understand - does this seem like a bug, or are you asking for clarification on how to use the desktop bank editor? Have you seen the [lesson videos](https://youtube.com/playlist?list=PL4a8Oe3qfS_-CefZFNYssT1kHdzEOdAlD)?

If this seems like a bug, can you be more specific about what you're trying to do that isn't working? When you say you can't add sounds, do you mean you can't create additional patches, or you can't use add different soundfonts or soundfont presets to a patch? By default, FluidPatcher on Windows saves your files to %HOMEPATH%\Documents\FluidPatcher (i.e. C:\Users\YourName\Documents\FluidPatcher) - if this directory is not writable for some reason that could be the issue.
